front/app/components/admin/Graphs/Statistic/StatisticBottomLabel.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { Box, Text } from '@citizenlab/cl2-component-library';
+
+interface Props {
+  bottomLabel: string;
+  bottomLabelValue?: string;
+}
+
+const StatisticBottomLabel = ({ bottomLabel, bottomLabelValue }: Props) => {
```

Comment:
I split up the `Statistic/index` component into sub components, to make it more modular. I thought this was better than adding too many props / conditionality